pkg/database/membership.go

```diff
@@ -59,3 +111,146 @@ func (m *Membership) Can(p rbac.Permission) bool {
 	}
 	return rbac.Can(m.Permissions, p)
 }
+
+// ListMemberships lists the memberships for this user. Use
+// ListMembershipsCached where possible.
+func (u *User) ListMemberships(db *Database) ([]*Membership, error) {
```

Comment:
Hmm... I find it weird that the user functions are in membership.go. I feel like the functions should be in the file for their pointer receiver.